infra/terraform/test-org/github/README.md

```diff
@@ -0,0 +1,13 @@
+# CFT GitHub
+
+## Operating
+
+This module requires setting a GitHub personal access token.
+
+First, create a [personal access token](https://help.github.com/en/enterprise/2.17/user/authenticating-to-github/creating-a-personal-access-token-for-the-command-line#creating-a-token).
+
+Then, export it as a Terraform variable:
+
+```
+export TF_VAR_github_token=aaaaaaa
```

Comment:
I suggest we stick with our standard for configuring credentials and just require [GITHUB_TOKEN](https://www.terraform.io/docs/providers/github/#token) to be defined in the environment.